### PR TITLE
New playbooks

### PR DESCRIPTION
**Pull request by dorsha**
@meirwah / @liorkol - please review.
These are the playbooks amir and slavik created with the new architecture.

### CODE REVIEW

**Comment by dorsha** [comment]
image

**Comment by dorsha** [comment]
Added to all playbooks:
```
version: -1
system: true
fromversion: 2.5.0
```

**Comment by dorsha** [comment]
Converted the phishing automated playbook as well.
@meirwah / @liorkol - if you don't want to review please let me know, I will merge the PR.

**Comment by slavikm** [comment]
Let's merge and open different PRs for the rewritten Phishing and others with all the new functionality.

File: Playbooks/playbook-XFE_Enrichment_Playbook.yml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+      }
+  "2":
+    id: "2"
+    taskid: 74769f79-7ed5-4b93-8169-acea2eafc627
+    type: regular
+    task:
+      id: 74769f79-7ed5-4b93-8169-acea2eafc627
+      version: -1
+      name: Check MD5s
+      script: xfe|||file
```
Comment (meirwah) [inline Playbooks/playbook-XFE_Enrichment_Playbook.yml]: xfe|||file ?? what does this mean?

**Comment by meirwah** [comment]
image

So what goes in here?
How does it not have any inputs?

**Comment by meirwah** [comment]
what:
``` file (crowdstrike-intel) ``` means?

File: Playbooks/playbook-XFE_Enrichment_Playbook.yml (hunk 1 above)
Comment (dorsha) [inline Playbooks/playbook-XFE_Enrichment_Playbook.yml]: This is the command id (must have the brand)

**Comment by dorsha** [comment]
The file is the command, the crowdstrike-intel is the name of the integration

**Comment by dorsha** [comment]
@meirwah 
The problem is that amir created the playbooks with the old non relevant integrations.. and you get warning about that (not exists)... will update them to the new ones.

image

**Comment by dorsha** [comment]
Fixed the old integration issue - good catch @meirwah 
@aviadl - please review my commit about that:
https://github.com/demisto/content/pull/312/commits/84be494d47054a70bd0fe2a542d59c58414dd53c

**Comment by aviadl** [comment]
@dorsha those changes looks fine

**Comment by dorsha** [comment]
Cool, merging.
Will add the new hunt and classifier in different branch once ready.

**Pull request merged**